OneSignalSDK/onesignal/build.gradle

```diff
@@ -55,6 +57,8 @@ dependencies {
     api 'androidx.browser:browser:[1.0.0, 1.99.99]'
     api 'androidx.appcompat:appcompat:[1.0.0, 1.99.99]'
     api 'androidx.work:work-runtime:[2.0.0, 2.99.99]'
+
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
Super nit picky here but lets use ' instead of " so its consistent with others

Reply:
oh good eye!

Reply:
oooh actually if we want to start using $kotlin_version variables on gradle, it only use it with " let mek nos if you prefer to continue with the old format

Reply:
Then maybe as we start replacing with `ext.` we will add `"` to all other dependencies